server/src/main/resources/db/V10__create_teams_table.sql

```diff
@@ -1,14 +1,15 @@
-DROP TABLE IF EXISTS team;
-CREATE TABLE team (
-  uuid varchar PRIMARY KEY,
-  name varchar,
-  description varchar
+drop table if exists teams;
+CREATE TABLE teams(
+   teamid varchar PRIMARY KEY,
+   name varchar UNIQUE,
+   description varchar
```

Comment:
This file can't be altered. You have to make a new migration file to make changes to a table. Also...rather than `teamid`, can we just go with `id`. I'd like to see this standardized across the project.

Reply:
added V18 alter script to update team/team_member table, updated teamid to uuid, some test issues if it was id.